platform/security/handler/security-handler-api/src/main/java/org/codice/ddf/security/handler/api/AnonymousAuthenticationToken.java

```diff
@@ -28,9 +31,20 @@
             BSTAuthenticationToken.BST_NS + BSTAuthenticationToken.TOKEN_VALUE_SEPARATOR
                     + BST_ANONYMOUS_LN;
 
-    public AnonymousAuthenticationToken(String realm) {
+    private String ipAddress;
```

Comment:
Made the anon credentials the ip address instead of keeping it in a member variable.